### ISSUE TITLE
Update prettifier library to 1.8.x

### ISSUE THREAD

**Issue by pstanislav** [victorporof/Sublime-HTMLPrettify#646]
Everything worked fine before (for the same project, same code), but since one of the recent updates some incorrect indentation happens in certain JavaScript files when this package is run to prettify.

I did some testing, and it appears to me that the problem occurs when async/await syntax is used for object methods, and it affects every such method of the object (except the last one), resulting in duplicated indent at the beginning of each method (e.g. 4 spaces instead of 2). Every such misalignment propagates through the rest of the object properties, creating cumulative effect. Only exception from the pattern, as already mentioned, is when such method is the last property of the object - indentation is correct then.

Example for code w/o async/await syntax (all ok):
```
module.exports = {
  first: function() {
    doSomethingSync();
    doSomethingElseSync();
  },
  second: function() {
    doSomethingCompletelyDifferentSync();
  },
  third: function() {
    doSomethingUnexpectedSync();
  },
};
```
Example for similar code w/ async/await syntax (issue):
```
module.exports = {
  first: async function() {
      await doSomething();
      await doSomethingElse();
    },
    second: async function() {
        await doSomethingCompletelyDifferent();
      },
      third: async function() {
        await doSomethingUnexpected(); // No issue for the last property
      },
};
```

**Comment by victorporof** [comment]
Please file this issue at https://github.com/beautify-web/js-beautify

**Comment by pstanislav** [comment]
Thanks. I see it was already reported [here](https://github.com/beautify-web/js-beautify/issues/1573) - fixed 4 days ago.

**Comment by bitwiseman** [comment]
@pstanislav @victorporof 
This fix is have been release in v1.8.8.

**Comment by victorporof** [comment]
Fixed https://github.com/victorporof/Sublime-HTMLPrettify/commit/db59fb56b7976ec4adb0546e78027690cadfc0dc
